nbextensions/codemirrormode/skill/README.md

```diff
@@ -0,0 +1,4 @@
+# CodeMirrorSkill
+
+This provides a *SKILL* mode for CodeMirror editor.
+The `skill` module adds a MIME type `x-skill` and a mode `skill` that can then be used with CodeMirror.
```

Comment:
Can you add a description/link of what skill is, so people don't have to google for it
(and if you search for skill alone you will never find the right answer...)